src/widget/wtracktableview.cpp

```diff
@@ -498,16 +482,15 @@ void WTrackTableView::createActions() {
         m_updateInExternalTrackCollections += updateInExternalTrackCollection;
         auto externalTrackCollectionPtr = updateInExternalTrackCollection.externalTrackCollection;
         connect(updateInExternalTrackCollection.action, &QAction::triggered,
-                [=](){
+                this, [=](){
```

Comment:
Do we have a preference or recommendation if we should pass captured parameters implicitly (like here) or explicitly (like in other occurrences)?

In Rust I would simply trust the compiler to do the right thing and avoid the redundant noise of explicit capturing. But I'm not aware what the C++ community is recommending.

Reply:
Actually there is a recommendation for this case:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rf-this-capture

Reply:
"For now, just be explicit."

Reply:
Originally, I didn't change the captured parameters because this is unrelated. I only added `this` because my linter complained ([`clazy-connect-3arg-lambda`](https://github.com/KDE/clazy/blob/master/docs/checks/README-connect-3arg-lambda.md)).

I changed it to explicit capturing now.